app/views/admin/users/_member_index.html.erb

```diff
@@ -1,10 +1,50 @@
 <div class="crayons-card p-3 s:p-4 m:p-7">
   <header class="flex flex-col l:flex-row justify-content-between l:items-center ">
-    <h1 class="crayons-title ml-3 l:ml-0">Members</h1>
+    <h1 class="crayons-title">Members</h1>
     <%= render "admin/users/index/tabs" %>
   </header>
     <%= render "admin/users/index/controls" %>
-    <table class="crayons-table" width="100%">
+
+    <!-- Small screen data view start -->
+    <div class="block m:hidden fs-s">
+      <h2 class="member-data-heading fs-s py-2 -mx-3 px-3 color-base-60">Members</h2>
+      <ul class="list-none">
+        <% @users.each do |user| %>
+          <li class="py-4">
+            <article>
+              <header class="flex">
+                <img class="radius-full mr-2" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="" />
+                <div class="flex flex-col">
+                  <h3 class="fs-base"><%= link_to user.name, admin_user_path(user) %></h3>
+                  <span class="color-base-60">@<%= user.username %></span>
+                </div>
+              </header>
+              <div class="flex justify-between py-2">
+                <div>standing info</div>
+                <div>role info</div>
+              </div>
+              <div class="flex justify-between items-end">
+                <dl class="flex gap-2">
```

Comment:
I love the DL-tag.  It's my favorite oft neglected HTML element.  And this looks to be the correct use of it!